README.md

```diff
@@ -818,9 +818,66 @@ CMD [ "java", "-Dgrpc-server.port=${GRPC_SERVER_PORT}", "-Dspring.datasource.url
 One more need was the storage space in the Kubernetes cluste to persist the data. This is specified through `pv.yaml` and `pvc.yaml` template files.
 
 
+**ConfigMaps**
 
+So far, application configuration is loaded at start-up (through environment variables) and cannot be modified without stoping the services affected.
 
+Kubernetes provides a resource called `ConfigMap` to provide the parameters to pass to a service in the form of key-value pairs.
 
+We wanted to explore the integration of this resource in our application and we decided to use **Spring Cloud Kubernetes Config** project. This framework/libray makes Kubernetes ConfigMap's available during application bootstrapping and, even more important, it triggers hot reloading of beans or Spring context when changes are produced in the ConfigMap.
+
+So, to take advantages of the Spring Cloud Kubernetes Config we introduced a new dependency in our pom.xml files:
+
+```xml
+<!--pom.xml-->
+
+       <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-kubernetes-config</artifactId>
+        </dependency>
+```
+
+It was also needed to include `bootstrap.properties` file in the services to indicate the name of the ConfigMap. If it is not specified, it will search for a ConfigMap witch the name of your Spring application.
```

Comment:
Actually, we are setting the name of the application in the 'bootstrap.properties' file and Spring Cloud Config automatically tries to find a configmap deployed in kubernetes with the name of the application.

Reply:
Agree.